wrappers/services_gen_test.go

```diff
@@ -263,3 +263,69 @@ func (s *servicesWrapperGenSuite) TestGenerateSnapServiceWithSockets(c *C) {
 	c.Assert(strings.Contains(string(generatedWrapper), "[Install]"), Equals, false)
 	c.Assert(strings.Contains(string(generatedWrapper), "WantedBy=multi-user.target"), Equals, false)
 }
+
+func (s *servicesWrapperGenSuite) TestServiceAfterBefore(c *C) {
+	const expectedServiceFmt = `[Unit]
+# Auto-generated, DO NOT EDIT
+Description=Service for snap application snap.app
+Requires=%s-snap-44.mount
+Wants=network-online.target
+After=%s-snap-44.mount network-online.target snap.snap.bar.service snap.snap.zed.service
+Before=snap.snap.foo.service
+X-Snappy=yes
+
+[Service]
+ExecStart=/usr/bin/snap run snap.app
+SyslogIdentifier=snap.app
+Restart=%s
+WorkingDirectory=/var/snap/snap/44
+
+
+
```

Comment:
Why so many spaces here?  If we're producing this, can we take the chance to fix it?